api/cache/names.go

```diff
@@ -32,23 +32,23 @@ func NewObjectsNameCache(config *Config) *ObjectsNameCache {
 	return &ObjectsNameCache{cache: gc}
 }
 
-// Get returns a cached object.
-func (o *ObjectsNameCache) Get(key string) *address.Address {
+// Get returns a cached object. Returns nil if value is missing.
+func (o *ObjectsNameCache) Get(key string) *oid.Address {
 	entry, err := o.cache.Get(key)
 	if err != nil {
 		return nil
 	}
 
-	result, ok := entry.(*address.Address)
+	result, ok := entry.(oid.Address)
```

Comment:
I suggest handling the conversion failure, otherwise the cache functionality will be completely broken and the program won't notice it. I like panic, but you can also log. Thoughts? @alexvanin @KirillovDenis

Reply:
I would prefer to log it. But we have to provide a logger in the caches. So I think we can do it in a separate issue.

Reply:
#465